src/appmixer/google/gmail/ListSignatures/component.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "appmixer.google.gmail.ListSignatures",
+    "author": "Sayam Nasir <sayam@client.io>",
+    "description": "Retrieves default signature from your Gmail.",
+    "private": true,
+    "version": "1.0.1",
+    "auth": {
+        "service": "appmixer:google",
+        "scope": ["https://www.googleapis.com/auth/gmail.settings.basic"]
```

Comment:
@sayam-nasir `gmail.settings.basic` is this scope needed? as it's a new scope and we don't have this approved, I'd require a re-validation of the scopes from Google. Could it be replaced by some existing scope?

Reply:
replaced it with the scope <https://www.googleapis.com/auth/gmail.readonly>